From 6f45fac549af94803eec6278b11b153d11d95d26 Mon Sep 17 00:00:00 2001
From: Simon van der Veldt <simon.vanderveldt@gmail.com>
Date: Tue, 6 Feb 2018 15:36:25 +0100
Subject: [PATCH 197/454] Add missing SND_PISOUND selects dependency to
 SND_RAWMIDI

Without it the Pisound module fails to compile.
See https://github.com/raspberrypi/linux/issues/2366

Signed-off-by: Simon van der Veldt <simon.vanderveldt@gmail.com>
---
 sound/soc/bcm/Kconfig | 1 +
 1 file changed, 1 insertion(+)

--- a/sound/soc/bcm/Kconfig
+++ b/sound/soc/bcm/Kconfig
@@ -193,5 +193,6 @@ config SND_BCM2708_SOC_FE_PI_AUDIO
 config SND_PISOUND
 	tristate "Support for Blokas Labs pisound"
 	depends on SND_BCM2708_SOC_I2S || SND_BCM2835_SOC_I2S
+	select SND_RAWMIDI
 	help
 	  Say Y or M if you want to add support for Blokas Labs pisound.
